src/rencache.c

```diff
@@ -28,7 +29,7 @@ typedef struct {
   RenColor color;
   RenFont *fonts[FONT_FALLBACK_MAX];
   float text_x;
-  char text[0];
+  char text[1];
```

Comment:
I think the actual syntax here (and if we're changing it here, we should change it in fonts too), is `char text[]`, which is probably what we should use.